tests/integration/init_tests.rs

```diff
@@ -6,7 +6,10 @@ use util::TestContextBuilder;
 
 #[test]
 fn init_subcommand_without_dir() {
-  let context = TestContextBuilder::new().use_temp_cwd().build();
+  let context = TestContextBuilder::new()
+    .env("JSR_URL", "https://jsr.io")
+    .use_temp_cwd()
+    .build();
```

Comment:
Not great, but I don't have a better idea how to test it properly. Maybe it's fine?

Reply:
Add an `@std/path` to the test server?

Reply:
I would have to dump serveral thousands lines of code into the test server 😬 is it worth it?

Reply:
It doesn't need to align with what's actually on the server. We can just do one file with a couple lines.